src/services/stag/StagAdapter.js

```diff
@@ -0,0 +1,39 @@
+const fetch = require('node-fetch');
+
+class StagAdapter {
+
+    /**
+     * @param {string} baseUrl
+     */
+    constructor(baseUrl) {
+        /** @type {string} */
+        this.baseUrl = baseUrl;
+    }
+
+    /**
+     * @param {string} operation
+     * @param {Object<string, string|number>} queryParams
+     * @return {Object} API call result
```

Comment:
```suggestion
     * @return {Promise} API call result
```
stačí bez typu  (v TypeScriptu by to bylo `Promise<any>`)